adventure.js

```diff
@@ -0,0 +1,54 @@
+var bar = alert ('Welcome to Tom & Jerry Pub order online make things easier for our servers');
+var age = prompt ('Please enter your age');
+if (age < 21) {
+	alert ('Sorry! please try some other time');
+} else {prompt ('What would you like to drink?', 'beer, hard-drink or cocktail?')
```

Comment:
You're not saving this `prompt` to a variable, which means you won't be able to use the response from the user in your code